python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         request_additional_params: Optional[Mapping[str, Any]] = None,
         log: logging.Logger = get_dagster_logger(),
         forward_logs: bool = True,
+        cancel_airbyte_sync_on_abnormal_termination: bool = True,
```

Comment:
I think we can remove `airbyte` from the param name since it's implicit from the resource, e.g. `cancel_sync_on_abnormal_termination`

Reply:
I believe this will also affect the behavior of manually cancelled runs via the UI, so maybe dropping the `abnormal` makes sense too? `cancel_sync_on_run_termination` seems reasonable

Reply:
I like this name much better - addressed in 1560855